app/models/concerns/tagging.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Tagging
```

Comment:
Purely cosmetic suggestion
```suggestion
module Taggable
```

Reply:
We can discuss this. "Concern tagging" sounded a little bit more natural than "concern taggable". Also it is concerning the taggings - or rather avoid obtaining the taggings.

But I don't know, we can change. It's not important.